packages/devtools_app/lib/src/screens/inspector/inspector_data_models.dart

```diff
@@ -439,62 +454,62 @@ class FlexLayoutProperties extends LayoutProperties {
   }
 
   Axis get crossAxisDirection {
-    return direction == Axis.horizontal ? Axis.vertical : Axis.horizontal;
+    return direction;
   }
 
-  double get mainAxisDimension => dimension(direction);
+  double? get mainAxisDimension => dimension(direction);
 
-  double get crossAxisDimension => dimension(crossAxisDirection);
+  double? get crossAxisDimension => dimension(crossAxisDirection);
```

Comment:
these can be made non-nullable if `size` above can be made non-nullable

Reply:
done